eng/source-build-patches/0004-Remove-Yarn-dependency-not-used-in-source-build.patch

```diff
@@ -0,0 +1,77 @@
+From 83c0f95c563fa6eb9b446f09166cac2c62ab6390 Mon Sep 17 00:00:00 2001
+From: dseefeld <dseefeld@microsoft.com>
+Date: Thu, 14 May 2020 16:29:28 +0000
+Subject: [PATCH 04/14] Remove Yarn dependency - not used in source-build
+
+---
+ eng/targets/Npm.Common.targets                                 | 3 ---
+ global.json                                                    | 1 -
+ .../src/Microsoft.Authentication.WebAssembly.Msal.csproj       | 2 --
+ ...oft.AspNetCore.Components.WebAssembly.Authentication.csproj | 2 --
+ src/Shared/E2ETesting/E2ETesting.targets                       | 3 ---
```

Comment:
I understand the global.json part of this patch but not why the other files (which shouldn't be involved in a source build) need to be touched. Could you explain❔

Reply:
I can't say with certainty, it may have been the result of a global search.